core/src/main/java/com/rudderstack/android/sdk/core/RudderClient.java

```diff
@@ -833,6 +842,32 @@ public interface Callback {
         void onReady(Object instance);
     }
 
+    /**
+     * Internal method
+     * @return instance of RudderUserSession
+     */
+    static RudderUserSession getUserSession() { return  userSession; }
```

Comment:
Why do we need getUserSession? Since all methods are already exposed through public APIs.